### PR TITLE
Please review and merge. Thank you!

### PR DESCRIPTION
**Pull request by NiyatiMPatel**

### CODE REVIEW

File: Art/niyati/index.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.container {
+  width: 100%;
+  overflow: hidden;
+  background: url(./image/sky.jpg);
```
Comment (LaurelineP) [inline Art/niyati/index.css:4]: ## Missing files
Missing files ( and folder image) in your folder:
- image/sky.jpg
- image/road.jpg
- image/city.png

File: Art/niyati/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <div class="city"></div>
+      <div class="car">
+        <img src="./image/car.png" alt="" />
+        <div class="wheel back">
+          <img src="./image/wheel3.png" alt="" />
+        </div>
+        <div class="wheel front">
+          <img src="./image/wheel3.png" alt="" />
+        </div>
+      </div>
```
Comment (LaurelineP) [inline Art/niyati/index.html:24]: ## Missing files
Missing files ( and folder image) in your folder:

- image/car.png
- image/wheel3.png

**Comment by LaurelineP** [comment]
Aloha @NiyatiMPatel 🙌
Thanks for your contribution and time !
Before we could merge your contribution, the following feedbacks need to be addressed first.
Please address the given feedbacks

> [!NOTE]
Checked values - are the one that are correctly done
> - Please address all the unchecked elements
> - If you need guidance , please refer to the

> [!TIP]
> If the name of your folder needs to be changed be sure to adjust the `include.js`

## 📌 Feedbacks:
- the foldername should be named after your unique github username
- After changing the foldername, double check for the lines you added in include.js as the path would change
- a bunch of missing file for your animation in the inexistant folder image ( see reviews ) , please add the folder and files referred in your code

## 🔹 Reference Acceptance criteria for a contribution
Correctly:
  - [ ] Folder name: named after the github user name in `./Art`  
     - [x] contains a `HTML` file: the contribution must have an HTML file
         - [x] has a <link> tag to the CSS file of the project
         - [x] contains the `HTML` content for the animation
         - [x] does not contain a SVG replacing the whole HTML CSS animation requirements
         - [x] does not include any CSS library:  No CSS library
         - [x] does not contain any JS script:  No JS logic should be part of your submission
     - [x] contains a `CSS` file: the contribution must have a CSS file 
     - [x] the contribution includes the animated file (gif) or picture ( jpeg, jpg, png )
     - [x] the pictural representation must not include the browser frame, just the animation
     - [x] the contribution includes the correct contribution's details in `include.js`
        - [ ]  the relative path should be correct : referring to the correct named folder
        - [x] `author`: the value should be the github name or github
        - [x] `artName`: the value should be the animation title
        - [ ] `pageLink`: the value should  be the relative path to your project's `HTML` file
        - [ ] `imageLink`: should refer to the project relative path's image 
        - [x] `githubLink`: the value should be your github profile link
        - [x] does not have chevrons
        - [x] does not have comments for each fields

**Review by NiyatiMPatel — COMMENTED** [review]
added missing files

**Review by NiyatiMPatel — COMMENTED** [review]
resolved conflict, accepted both current and incoming changes.

**Pull request merged**